crates/symbolicator-proguard/src/symbolication.rs

```diff
@@ -216,6 +216,11 @@ impl ProguardService {
                     )
                 })
             })
+            // This is for parity with the Python implementation. It's unclear why remapping a frame with line 0
+            // would produce useful information, and I have no conclusive evidence that it does.
+            // See the `test_line_0` and `test_line_0_2` integration tests for examples of the results this produces.
+            //
+            // TODO(@loewenheim): Find out if this is useful and remove it otherwise.
```

Comment:
Can you also link to the pull-request here? Since we're going to remove the python implementation, it would be super hard to find the original pull-request.